geth/node/node.go

```diff
@@ -269,18 +269,30 @@ func makeIPCPath(config *params.NodeConfig) string {
 
 // parseNodes creates list of discover.Node out of enode strings.
 func parseNodes(enodes []string) []*discover.Node {
-	nodes := make([]*discover.Node, len(enodes))
-	for i, enode := range enodes {
-		nodes[i] = discover.MustParseNode(enode)
+	var nodes []*discover.Node
+	for _, enode := range enodes {
+		parsedPeer, err := discover.ParseNode(enode)
+		if err == nil {
+			nodes = append(nodes, parsedPeer)
+		} else {
+			logger.Warn("Failed to parse enode", "enode", enode, "err", err)
```

Comment:
i would put Error

Reply:
Shouldn't we somehow communicate that back to status-react? A user will type in an invalid enode address and she/he will be convinced that it worked while it silently failed. 

I looked into the code and it may require some more refactoring of the params so we can create a separate issue to fix that.

Reply:
@adambabik I agree, but as far as `status-react` is concerned is ok to fail silently for now, as this is only available through advanced menu, I think eventually we might want to signal that though.